src/index.html

```diff
@@ -10,6 +12,21 @@ <h2>How many do you know?</h2>
       <input type="text" id="input1" title="input1" />
       <button type="submit">Submit</button>
     </form>
+    <div>
+        <div id="progress-live" aria-live="polite" class="visually-hidden"></div>
+        <label for="progress-bar">
+            <span aria-hidden="true">Your Progress: </span>
+            <span id="progress-description" class="visually-hidden">Your progress: 0 out of 124 tags added</span>
+        </label>
+        <progress 
+            id="progress-bar" 
+            aria-valuemin="0"
+            aria-valuemax="100"
+            aria-valuetext="0"
```

Comment:
Do we even need these aria values? Progress attributes like  `min` and `max` seem to work on MacOS Safari VoiceOver just fine, since we are using the native HTML tag.